<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trainer</class>
 <widget class="QWidget" name="trainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>591</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="leFio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePhone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twTrainerA">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teTrainerA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfoTrA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPhoneTrA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телефона</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFIOTrA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddTrA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelTrA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAllTrA">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackTrA">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключеие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тренеры</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
